### DISCUSSION TITLE
Context variables in notifications templates

### DISCUSSION THREAD

**Discussion by pshrm** [argoproj/argo-rollouts#3356] [Q&A]
I see that you could define context variables in argocd map and they are available in argocd notifications to be used.
e.g., https://github.com/argoproj-labs/argocd-notifications/discussions/169
is there a similar way to do same for argo rollouts?

**Comment by zachaller** [comment]
ArgoCD and Rollouts use the same notification-engine library so yes it should work.

**Reply by pshrm** [reply]
Hi, 
No, it does not. I have tested it
